src/Stream.php

```diff
@@ -60,7 +60,7 @@ public function __construct($stream, $mode = 'r')
     public function __toString()
     {
         if (! $this->isReadable()) {
-            return '';
+            throw new RuntimeException('Stream is not readable');
```

Comment:
Ah, now I remember the horror of debugging ZF1's view helper that throwing exception...